docusaurus/docs/Android/04-compose/04-message-components/01-messages-screen.mdx

```diff
@@ -5,7 +5,7 @@ The `MessagesScreen` component is the second screen component in the SDK, which
 * [`MessageListHeader`](./02-message-list-header.mdx): Displays a back button, the name of the channel or thread and a channel avatar.
 * [`MessageList`](./03-message-list.mdx): Loads messages data and shows different states based on the data such as loading, empty and loaded states. Also sets up various action handlers, such as long and single item taps, handles pagination, and provides an option to scroll to the bottom.
 * [`MessageComposer`](./04-message-composer.mdx): Sets up and handles the message input, as well as attachments and message actions like editing and replying.
-* [`SelectedMessageOverlay`](./06-selected-message-overlay.mdx): Shown when the user long taps and selects a message. Allows the user to react to messages and perform different actions, like deleting, editing, replying, starting a thread and more.
+* [`SelectedMessageMenu`](./06-selected-message-menu.mdx): Shown when the user long taps and selects a message. Allows the user to react to messages and perform different actions, like deleting, editing, replying, starting a thread and more.
```

Comment:
Do you think we need to add also docs for `SelectedReactionsMenu`?

Reply:
We definitely should, since it'll probably be commonly used.